apps/prairielearn/src/ee/models/course-instance-required-plans.ts

```diff
@@ -0,0 +1,55 @@
+import { loadSqlEquiv, queryRow } from '@prairielearn/postgres';
+import { PlanName } from '../lib/billing/plans-types';
+import { CourseInstanceRequiredPlanSchema } from '../../lib/db-types';
+import { insertAuditLog } from '../../models/audit-log';
+
+const sql = loadSqlEquiv(__filename);
+
+export async function insertCourseInstanceRequiredPlan(
+  course_instance_id: string,
+  plan: PlanName,
+  authn_user_id: string,
+) {
+  const newRequiredPlan = await queryRow(
+    sql.insert_required_plan_for_course_instance,
+    { course_instance_id, plan_name: plan },
+    CourseInstanceRequiredPlanSchema,
+  );
+  await insertAuditLog({
+    authn_user_id,
+    table_name: 'course_instance_required_plans',
+    action: 'insert',
+    institution_id: null, // TODO: get value for this
+    course_id: null, // TODO: get value for this
```

Comment:
Would be nice to have `insertAuditLog` handle this for us, like it does in PT. @mwest1066 do you have (or could you make) a non-sproc version of that in mind that we could use in PL?

Reply:
I haven't yet made a non-sproc version of this but I plan to do so soon.

Reply:
I took the "do this the stupid way so we can ship something" approach in b4d0aa9.

Reply:
lgtm